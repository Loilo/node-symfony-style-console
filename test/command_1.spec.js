

const wrapper = require('./wrapper')

test('01 Ensure has single blank line between titles and blocks', wrapper(io => {
	io.title('Title')
	io.warning('Lorem ipsum dolor sit amet')
	io.title('Title')
}))
