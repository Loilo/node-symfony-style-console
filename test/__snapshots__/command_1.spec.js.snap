// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`01 Ensure has single blank line between titles and blocks 1`] = `
"Title
=====

 [WARNING] Lorem ipsum dolor sit amet                                           

Title
====="
`;
